site/content/docs/developing/environment-variables.en.md

```diff
@@ -56,6 +56,22 @@ Here's a quick guide showing you how to add environment variables to your app by
 
 ![Editing the manifest to add env vars](https://raw.githubusercontent.com/kohidave/ecs-cliv2-demos/master/env-vars-edit.svg?sanitize=true)
 
+Additionally, if you want to specify environment variables in bulk, you can add them in your [env file](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/taskdef-envfiles.html#taskdef-envfiles-considerations) and pass its location to your [manifest](../manifest/overview.en.md) in the `env_file` section.
+
+```yaml
+# in copilot/{service name}/manifest.yml
+env_file: log.env
+```
+
+And in `log.env` we could have
+```
+#This is a comment and will be ignored
+LOG_LEVEL=debug
+```
+
+!!! Attention
+    You can only specify one env file in each of your workspace.
```

Comment:
```suggestion
    You can specify only one env file per workspace.
```

Reply:
Do we need this attention block? won't it error if they specify a list when unmarshaling the manifest?

Reply:
Yeah that's true. I'll remove this block.